zoo/policies/__init__.py

```diff
@@ -100,3 +103,65 @@ def human_keyboard_entrypoint(*arg, **kwargs):
 
 
 register(locator="human-in-the-loop-v0", entry_point=human_keyboard_entrypoint)
+
+
+from smarts.env.multi_scenario_env import resolve_agent_interface
+
+
+def load_config(path):
+    import yaml
+
+    config = None
+    if path.exists():
+        with open(path, "r") as file:
+            config = yaml.safe_load(file)
+    return config
+
+
+root_path = str(Path(__file__).absolute().parents[2])
+config = load_config(
+    Path(os.path.join(root_path, "competition/track1/submission/config.yaml"))
```

Comment:
This path should be specified in the entrypoint using the `policy_path` that was passed in. (i.e. `config = load_config(policy_path + "/config.yaml"`)